Code/Editor/Dialogs/PythonScriptsDialog.h

```diff
@@ -45,5 +46,5 @@ private slots:
     void ScanFolderForScripts(QString path, AZStd::vector<QString>& scriptFolders) const;
 
 private:
-    QScopedPointer<Ui::CPythonScriptsDialog> ui;
```

Comment:
Are we really sure about this?

https://doc.qt.io/qt-6/qscopedpointer.html indicates that its fine to use forward declared scoped pointers as long as the destructor of the class is not inline and the same goes for qt5 (I dug up the docs).

What compile error does this actually cause?

Reply:
Actually you're right thanks for checking ! Updated the Pr